inc/session_handler/class-disallowed-session-handler.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Altis Cloud Session handler.
+ *
+ * @package altis/cloud
+ */
+
+namespace Altis\Cloud\Session_Handler;
+
+// phpcs:disable Generic.Commenting.DocComment.MissingShort
+// phpcs:disable Squiz.Commenting.FunctionComment.MissingParamTag
+// phpcs:disable Squiz.Commenting.FunctionComment.Missing
+
+/**
+ * Disallowed Session handler class.
+ *
+ * Disables creation of sessions via short-circuiting session_open() calls.
+ */
+class Disallowed_Session_Handler implements \SessionHandlerInterface {
+	function open( $save_path, $session_name ) {
+		error_log( 'Sessions are not enabled in Altis config.' );
```

Comment:
Use `trigger_error( '...', E_USER_WARNING );` instead. `error_log()` just writes to log file but doesn't raise any kind of warning or error and doesn't have a severity level. Using `trigger_error()` means tools like Query Monitor will pick up and show the warnings.